### ISSUE TITLE
file-header rule doesn't correctly parse single line comments

### ISSUE THREAD

**Issue by battmanz** [palantir/tslint#2319]
### Bug Report

- __TSLint version__: 4.5.1
- __TypeScript version__: 2.0.10
- __Running TSLint via__: CLI

#### TypeScript code being linted

```ts
// Bad Header

export class Foo {
}
```

with `tslint.json` configuration:

```json
{
  "rules": [
    "file-header": [true, "Copyright \\d{4}"]
  ]
}
```

#### Actual behavior
There's no warning

#### Expected behavior
There should be a warning.
